styleguide-app/Debug/Control/View.elm

```diff
@@ -137,7 +137,20 @@ viewSection name children =
 
 
 codeFromList : List ( String, a ) -> String
-codeFromList list =
-    "\n\t[ "
-        ++ String.join "\n\t, " (List.map Tuple.first list)
-        ++ "\n\t] "
+codeFromList =
+    codeFromListWithIndentLevel 1
+
+
+codeFromListWithIndentLevel : Int -> List ( String, a ) -> String
+codeFromListWithIndentLevel indent list =
+    let
+        indents =
+            String.repeat indent "\t"
```

Comment:
This leaves us with invalid elm if we want to copy, but that's ok right?

Reply:
Thanks for this note! I have the tabs replaced in the Ellie version, but I think you're right that I should replace them in the rendered examples too.